Lab_13_Make_Money/script.js

```diff
@@ -0,0 +1,57 @@
+// get the form from HTML
+const moneyForm = document.getElementById("money-form")
+// get coin display div from HTML
+const coinDisplay = document.getElementById("coin-display")
+
+// add event listener so DOM knows what to do at submission
+moneyForm.addEventListener("submit", e => {
+    // prevent default submission behavior
+    e.preventDefault()
+    // at submission, get values for coin type and coin count from the form
+    const coinType = document.getElementById("coin-type").value
+    const coinCount = parseInt(document.getElementById("coin-count").value)
+
+    // testing out the button, not getting any results
+    /* 
+    const makeMoneyButton = document.querySelector("button")
+    makeMoneyButton.addEventListener("click", () => {
+        console.log(coinType)
+    }) 
+    */
+
+    // call functions below
+    hideCoinsAtStart()
+    displayCoins(coinType, coinCount)
```

Comment:
See below for how this approach would work

```suggestion
    addCoins(coinType, coinCount)
```